docs/docs/sources/facebook.md

```diff
@@ -5,6 +5,22 @@ sidebar_label: Facebook
 
 The Facebook source provides a channel of communication between a Facebook page and your running instance of the Airy Core Platform.
 
+## Connecting a Facebook Page
+
+To connect a page, the first requirement is to have an approved app by Facebook. If you don't have one, you will need to create it.
+
+After creating it, and Facebook approves it you should see it in https://developers.facebook.com/apps/
+
+![Facebook Apps](apps.png)
```

Comment:
Nitpick: It'd be great to use `jpg`, because of how well they compress on the web. (I know screenshots get stored as a png by default)